logstash/class-logger.php

```diff
@@ -306,13 +306,14 @@ protected static function parse_params( array $params ) : array {
 		];
 
 		if ( ! isset( $params['file'] ) && ! isset( $params['line'] ) ) {
+			// phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_debug_backtrace
 			$backtrace = debug_backtrace( DEBUG_BACKTRACE_IGNORE_ARGS, 1 );
 
-			if ( isset( $backtrace[0] ) && isset( $backtrace[0]['file'] ) ) {
+			if ( isset( $backtrace[0]['file'] ) ) {
```

Comment:
Here and below: the second `isset()` checks both `$backtrace[0]` and `$backtrace[0]['file']`; the first `isset()` check is superfluous.